test/curves/test_ushape.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pandas as pd
+from openest.curves.basic import ZeroInterceptPolynomialCurve, ClippedCurve, ShiftedCurve, MinimumCurve
+from openest.curves import ushape_numeric, ushape_analytic
+
+# Read polymins
+polymins = pd.read_csv("polymins.csv")
+allcalcs = pd.read_csv("allcalcs.csv")
+
+tas = np.arange(-40, 50, .01)
+fillins = np.arange(-40, 50, .1)
+    
+for ii in range(polymins.shape[0]):
+    print ii
```

Comment:
I imagine this was aiming for `print(ii)`.

Reply:
Fixed.